backend-zs/routes/story.routes.js

```diff
@@ -0,0 +1,11 @@
+module.exports = storyRoutes;
+
+function storyRoutes() {
+    var storiesController = require('../controllers/stories.controller');
+    var router = require('express').Router();
+
+    router.get('/all', storiesController.getStories);
```

Comment:
We can use a middleware for validating request data
https://express-validator.github.io/docs/

Reply:
I think it's a good idea, but not relevant to get requests

Reply:
It depends, whether everyone is allowed to view the stories or only logged in users (in this case user data would be needed in the request body and thus validated)

Reply:
Not really sure whether or not we discussed this with Felix

Reply:
okay, let's assume it's not needed for now.